bitnami/argo-cd/templates/repo-server/deployment.yaml

```diff
@@ -137,6 +137,42 @@ spec:
                   key: {{ include "argocd.redis.secretPasswordKey" . }}
           {{- end }}
         {{- end }}
+        {{- if .Values.repoServer.configManagementPlugins.enabled }}
+        - name: copy-cmp-server-binary
+          image: {{ include "argocd.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - /bin/cp
+            - -n
+            - /opt/bitnami/argo-cd/bin/argocd
+            - /additional-binaries/argocd-cmp-server
+          {{- if .Values.repoServer.containerSecurityContext.enabled }}
+          securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" .Values.repoServer.containerSecurityContext "context" $) | nindent 12 }}
+          {{- end }}
+          volumeMounts:
+            - mountPath: /additional-binaries
+              name: cmp-additional-binaries
+        {{- with .Values.repoServer.configManagementPlugins.additionalBinaries }}
+        {{- if .binaries }}
+        - name: download-additional-binaries
+          image: {{ include "argocd.repo-server.config-management-plugins.additional-binaries.image" $ }}
+          imagePullPolicy: {{ .image.pullPolicy }}
+          command:
+            - sh
+            - -ec
+          args:
+            - |-
+              {{- range $additionalBinary := .binaries }}
+              # downloading {{ $additionalBinary.name }}
+              curl -L {{ $additionalBinary.url }} -o /additional-binaries/{{ $additionalBinary.name }}
+              chmod +x /additional-binaries/{{ $additionalBinary.name }}
+              {{- end }}
+          volumeMounts:
+            - mountPath: /additional-binaries
+              name: cmp-additional-binaries
```

Comment:
These two steps look overcomplicated to me. I guess we're using `bitnami/os-shell` to download extra binaries because `bitnami/argo-cd` lacks `curl`, right?